python/large_test_files.py

```diff
@@ -49,11 +76,14 @@ def create_file(self, row_count=None):
         self.bank_name = self.ts_df.iloc[0][1]
         
         #Stores the second column of the LAR data as "lei."    
```

Comment:
This code looks like it is pulling the 14th column of the TS not the second of the LAR, is the comment out date?

Reply:
Yes, thanks for finding!